hw/ip/lc_ctrl/doc/_index.md

```diff
@@ -195,41 +200,220 @@ This ensures that during specific states (RAW, TEST_LOCKED, SCRAP, INVALID) it i
+
+- *_TOKEN
+- CREATOR_ROOT_KEY*
+
+Specifically this means after OTP sensing, the above entries are unwrapped to obtain the real value.
+Similarly, during programming, they are wrapped before beginning to be written to OTP.
+
+The function used for this wrapping is the lightweight PRESENT-cipher.
+The wrapping is a one time event during controlled manufacturing, and unwrapping also cannot be supplied with arbitrary ciphertexts.
+Thus the system cannot be abused to generate a large number of traces for informational leakage, and thus a fully hardened cipher (such as AES) is not required.
+
+Note also, a global key is used here because there is no other non-volatile location to store a secret key.
+If PUFs were available (either in memory form or fused form), it could become an appealing alternative to hold a device unique fuse key.
+
+See the [OTP controller]({{< relref "hw/ip/otp_ctrl/doc/_index.md#secret-vs-non-secret-partitions">}}) for more details.
+
+### OTP Accessibility Summary and Impact of Life Cycle Signals
+
+A subset of secret collateral is further access-controlled by the life cycle CREATOR_SEED_SW_RW_EN signal.
+These are
+
+- RMA_UNLOCK_TOKEN
+- CREATOR_ROOT_KEY
+
+The table below summarizes the software accessibility of all life cycle collateral.
+
+{{< snippet "hw/ip/lc_ctrl/doc/lc_ctrl_otp_accessibility.md" >}}
+
+Note that CREATOR_SEED_SW_RW_EN is set to OFF if SECRET2_DIGEST has a nonzero value in PROD, PROD_END and DEV states.
+SEED_HW_RD_EN only becomes active if SECRET2_DIGEST has a nonzero value in DEV, PROD, PROD_END and RMA states.
+
+## Flash Collateral
+
+The flash contains both memory mapped and non-memory mapped partitions.
+As it pertains to life cycle, the flash contains three sets of important collateral.
+They are enumerated in the table below.
+Just as with OTP, the consumer and usage of each is also described.
+
+{{< snippet "hw/ip/lc_ctrl/doc/lc_ctrl_flash_collateral.md" >}}
+
+Each collateral belongs to a separate flash partition, the table below enumerates the partition and whether the partition is memory mapped.
+
+{{< snippet "hw/ip/lc_ctrl/doc/lc_ctrl_flash_partitions.md" >}}
+
+The general flash partition refers to any software managed storage in flash, and is not a specific carve out in the non-memory mapped area.
+
+### Flash Accessibility Summary and Impact of Life Cycle Signals
+
+The creator software is trusted to manage the owner partition (OWNER_DATA).
+As such, OWNER_DATA is remains accessible during DEV / PROD / PROD_END / RMA states, irrespective of the device personalization state.
+It is expected that ROM_ext during secure boot programs the protection correctly such that downstream software has appropriate permissions.
+
+The CREATOR_DATA partitions however, are further qualified based on the personalization state of the device.
+Just as with OTP, the table below enumerates accessibility of flash collateral.
+
+{{< snippet "hw/ip/lc_ctrl/doc/lc_ctrl_flash_accessibility.md" >}}
```

Comment:
same comment as above, the "blob" no longer exists.
I should also think about how to word this on flash side.
Maybe I should describe it more as to how it links to partitions and provide a reference here?

Reply:
ok let me remove. do you want to insert a plug pointing to the flash docs here? or vice versa?

Reply:
let me work it out on flash side first and i'll come back and insert a link.
I think whatever you have is good, I just wanted to add some more language on how these partitions actually link to flash's data and info partition types.  

So that probably belongs in flash.

Reply:
sorry just to clarify, DON'T remove what you have!
i just meant i'll add another link later.

Reply:
ok sounds good!